### PR TITLE
Fixed typos and spelling mistakes in chapters 5 and 6

### PR DESCRIPTION
**Pull request by pelegs**
In addition, the text constantly jumps between "antialias" and "anti-alias". I looked at Wikipedia and it seems that "anti-alias" is the commonly used form, so I changed all the occurrences in the text to conform (while leaving links and image names unchanged).

### CODE REVIEW

File: code/chapter-06/circle-sdf-distances.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
     varying vec2 v_position;
     void main()
     {
-        const float epsilon = 0.005;
+        const float epsilon = 0.05;
```
Comment (rougier) [inline code/chapter-06/circle-sdf-distances.py]: Why did you change this value ?

File: code/chapter-05/hollow-cube.py [hunk 2]
```diff
@@ -32,9 +32,9 @@
     float xy = min( abs(v_position.x), abs(v_position.y));
     float xz = min( abs(v_position.x), abs(v_position.z));
     float yz = min( abs(v_position.y), abs(v_position.z));
-    float b1 = 0.74;
-    float b2 = 0.76;
-    float b3 = 0.98;
+    float b1 = 0.7;
+    float b2 = 0.75;
+    float b3 = 0.95;
```
Comment (rougier) [inline code/chapter-05/hollow-cube.py]: Any reason to change these values?

File: code/chapter-05/border-cube.py [hunk 3]
```diff
@@ -32,7 +32,7 @@
     float xy = min( abs(v_position.x), abs(v_position.y));
     float xz = min( abs(v_position.x), abs(v_position.z));
     float yz = min( abs(v_position.y), abs(v_position.z));
-    float b = 0.85;
+    float b = 0.95;
```
Comment (rougier) [inline code/chapter-05/border-cube.py]: Any reason to change this value ?

File: 05-cube.rst [hunk 4]
```diff
@@ -370,13 +370,13 @@ coordinates can be pretty much anything but for the sake of simplicity, we'll
 stick to the [0,1] range). Since we are displaying a cube, we'll use one
 texture per side and the texture coordinates are quite easy to define: [0,0],
 [0,1], [1,0] and [1,1]. Of course, we have to take care of assigning the right
-texture coordinates to the right vertex or you texture will be messed up.
+texture coordinates to the right vertex or the texture will be messed up.
 
-Furthemore, we'll need some extra work because we cannot share anymore our
-vertices between faces since they won't share their texture coordinates. We
+Furthemore, we'll need some extra work because we cannot share our
```
Comment (rougier) [inline 05-cube.rst]: Furthemore -> Furthermore

File: code/chapter-06/circle-sdf-distances.py (hunk 1 above)
Comment (pelegs) [reply code/chapter-06/circle-sdf-distances.py]: Oh... I was playing with the values when following your tutorial (just to see how stuff change), and probably forgot to return them to the original. You can freely ignore these kind of changes.
My bad.

Comment (rougier) [reply code/chapter-06/circle-sdf-distances.py]: Can you put back original values such that I can merge your PR ?

**Comment by rougier** [comment]
Thanks!
